autopilot/agent.go

```diff
@@ -229,15 +235,11 @@ func (a *Agent) OnBalanceChange(delta btcutil.Amount) {
 // OnNodeUpdates is a callback that should be executed each time our channel
 // graph has new nodes or their node announcements are updated.
 func (a *Agent) OnNodeUpdates() {
-	a.wg.Add(1)
-	go func() {
-		defer a.wg.Done()
 
-		select {
-		case a.stateUpdates <- &nodeUpdates{}:
-		case <-a.quit:
-		}
-	}()
+	select {
+	case a.nodeUpdates <- &nodeUpdates{}:
+	default:
```

Comment:
Should also read `quit`?

Reply:
Not needed as default will be selected immediately if the `nodeUpdates` channel is full.